include/drives.h

```diff
@@ -124,6 +124,7 @@ class localDrive : public DOS_Drive {
 
 private:
 	bool IsFirstEncounter(const std::string& filename);
+	bool FileIsReadOnly(char* name);
```

Comment:
const char* ?

Reply:
It was calling into another function that wasn't using const.  I pushed a commit to just `const` all the `char *` arguments in these classes.  Luckily it just compiled so no one was modifying the strings in place.